ghwatch.py

```diff
@@ -195,7 +195,17 @@ def get_html_url(ghbase, rec):
 def priority_sort_key(item):
     return (-REASON_PRIO.index(item.reason), item.updated_at)
 
-def draw():
+def github_load(user):
+    since = datetime.datetime.utcnow() - datetime.timedelta(days=args.days)
+    get_all = True if args.all else GithubObject.NotSet
+    issues = list(user.get_notifications(all=get_all, since=since))
+
+    if sort_notifications:
+        issues.sort(key=priority_sort_key, reverse=True)
+
```

Comment:
not sure this is necessary but fwiw the draw could be even faster if you move the ` ghmeta.get` into here as well, then put it in the issue like `issue['ghmeta']` to use it in draw